### DISCUSSION TITLE
how to use third party react component in feliz

### DISCUSSION THREAD

**Discussion by mohaalak** [Zaid-Ajaj/Feliz#315] [Q&A]
for fable react this [page](https://github.com/fable-compiler/fable-react/blob/master/docs/using-third-party-react-components.md#using-third-party-react-components) show how to use third party component,
but I don't know how to write the third party react component for Feliz. I think the documentation for this is missing.

**Comment by 64J0** [comment]
Hey @mohaalak, I think that [this page](https://safe-stack.github.io/docs/recipes/javascript/third-party-react-package/) is what you're looking for. Check the last topic there, it presents an example on how to Add Support for a Third Party React Library using Feliz.

**Comment by laurentpayot** [comment]
@mohaalak _If I’m right_, you can use third party React components the same way Feliz docs explain how to consume JSX components: https://zaid-ajaj.github.io/Feliz/#/Feliz/UsingJsx
